readthedocs/doc_builder/python_environments.py

```diff
@@ -223,6 +223,24 @@ def setup_base(self):
 
     def install_core_requirements(self):
         """Install basic Read the Docs requirements into the virtualenv."""
+        pip_install_cmd = [
+            'python',
+            self.venv_bin(filename='pip'),
+            'install',
+            '--upgrade',
+            '--cache-dir',
+            self.project.pip_cache_path,
+        ]
+
+        # Install latest pip first,
+        # so it is used when installing the other requirements.
+        cmd = pip_install_cmd + ['pip==18.1']
```

Comment:
Do we really expect to update this regularly? Feels like we should probably just call it with `--upgrade`, but I also worry about being exposed to pip bugs on release :/

Reply:
Well, the release history looks good https://pypi.org/project/pip/#history. Not sure, I'm also worry about not having control of the version in case a bug comes up from pip.

Reply:
`pip` is a really mature piece of software with active development. All the python ecosystem relies on it. I think it will be better to just unpin it and rely on them.

The case would be different if we are talking about a different package.